Sources/MediaCategories/MediaCategoryViewController.swift

```diff
@@ -291,11 +291,6 @@ class MediaCategoryViewController: UICollectionViewController, UISearchBarDelega
     @objc func themeDidChange() {
         collectionView?.backgroundColor = PresentationTheme.current.colors.background
         searchBar.backgroundColor = PresentationTheme.current.colors.background
-
-        if #available(iOS 13.0, *) {
```

Comment:
now navigationBar changes color without this code (cause of extension) - and without lag!